app/src/main/java/org/dhis2/data/forms/dataentry/tablefields/edittext/EditTextCellCustomHolder.java

```diff
@@ -111,6 +111,7 @@ public void update(@NonNull FieldViewModel model) {
     private void setInputType(ValueType valueType) {
 
         editText.setFilters(new InputFilter[]{});
+        editText.setFocusable(true);
```

Comment:
Do we need to take into account if the field is editable or not? Also, isn't this being called in the update method from the holder?

Reply:
Nope, in the update method it sets the editText from enabled to disabled. But focusable is not updated.